fuel-client/src/main.rs

```diff
@@ -64,6 +64,7 @@ impl CliArgs {
     }
 }
 
-fn main() {
+#[tokio::main]
+async fn main() {
     futures::executor::block_on(CliArgs::parse().exec());
```

Comment:
```suggestion
    CliArgs::parse().exec().await;
```

Is also needed for my changes to work